4.0 Crossref Websearch fails with "Parser error"

JabRef version 
```
JabRef 4.0
Linux 4.12.12-gentoo+ amd64 
Java 1.8.0_152
```

Steps to reproduce:
search for 10.1103/PhysRevLett.119.177201
![selection_369](https://user-images.githubusercontent.com/298682/32225308-f9e659ac-be44-11e7-905d-9849edc8fe66.png)

![selection_368](https://user-images.githubusercontent.com/298682/32225318-06442a94-be45-11e7-9556-d568ddf263be.png)
```

14:11:58.427 [JabRef CachedThreadPool] ERROR org.jabref.gui.importer.fetcher.SearchBasedEntryFetcher - Error while fetching from Crossref
org.jabref.logic.importer.FetcherException: An internal parser error occurred
	at org.jabref.logic.importer.SearchBasedParserFetcher.performSearch(SearchBasedParserFetcher.java:72) ~[JabRef-4.0.jar:?]
	at org.jabref.gui.importer.fetcher.SearchBasedEntryFetcher.processQuery(SearchBasedEntryFetcher.java:37) ~[JabRef-4.0.jar:?]
	at org.jabref.gui.importer.fetcher.GeneralFetcher.lambda$actionPerformed$5(GeneralFetcher.java:216) ~[JabRef-4.0.jar:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_152]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_152]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_152]
Caused by: org.jabref.logic.importer.ParseException: CrossRef API JSON format has changed
	at org.jabref.logic.importer.fetcher.CrossRef.jsonItemToBibEntry(CrossRef.java:131) ~[JabRef-4.0.jar:?]
	at org.jabref.logic.importer.fetcher.CrossRef.lambda$getParser$3(CrossRef.java:88) ~[JabRef-4.0.jar:?]
	at org.jabref.logic.importer.SearchBasedParserFetcher.performSearch(SearchBasedParserFetcher.java:60) ~[JabRef-4.0.jar:?]
	... 5 more
Caused by: org.json.JSONException: JSONObject["title"] not found.
	at org.json.JSONObject.get(JSONObject.java:471) ~[JabRef-4.0.jar:?]
	at org.json.JSONObject.getJSONArray(JSONObject.java:618) ~[JabRef-4.0.jar:?]
	at org.jabref.logic.importer.fetcher.CrossRef.jsonItemToBibEntry(CrossRef.java:113) ~[JabRef-4.0.jar:?]
	at org.jabref.logic.importer.fetcher.CrossRef.lambda$getParser$3(CrossRef.java:88) ~[JabRef-4.0.jar:?]
	at org.jabref.logic.importer.SearchBasedParserFetcher.performSearch(SearchBasedParserFetcher.java:60) ~[JabRef-4.0.jar:?]
	... 5 more

```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -37,6 +37,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
  - We fixed an issue where JabRef would not show the translated content at some points, although there existed a translation
  - We fixed an issue where editing in the source tab would override content of other entries [#3352](https://github.com/JabRef/jabref/issues/3352#issue-268580818)
  - We fixed several issues with the automatic linking of files in the entry editor where files were not found or not correctly saved in the bibtex source [#3346](https://github.com/JabRef/jabref/issues/3346)
+ - We fixed an issue where fetching entries from crossref that had no titles caused an error [#3376](https://github.com/JabRef/jabref/issues/3376)
 
  ### Removed
 

--- a/src/main/java/org/jabref/logic/importer/fetcher/CrossRef.java
+++ b/src/main/java/org/jabref/logic/importer/fetcher/CrossRef.java
@@ -110,7 +110,9 @@ public class CrossRef implements IdParserFetcher<DOI>, EntryBasedParserFetcher, 
         try {
             BibEntry entry = new BibEntry();
             entry.setType(convertType(item.getString("type")));
-            entry.setField(FieldName.TITLE, item.getJSONArray("title").optString(0));
+            entry.setField(FieldName.TITLE,
+                    Optional.ofNullable(item.optJSONArray("title"))
+                            .map(array -> array.optString(0)).orElse(""));
             entry.setField(FieldName.SUBTITLE,
                     Optional.ofNullable(item.optJSONArray("subtitle"))
                             .map(array -> array.optString(0)).orElse(""));

